app/queue.py

```diff
@@ -112,6 +121,11 @@ def __move_to_inflight(self, in_flight_key: str, count: int) -> list[str]:
         decoded = [result.decode("utf-8") for result in results]
         return decoded
 
+    def move_from_inflight(self, in_flight_key: str):
```

Comment:
Just to indicate this is an internal API to the queue:

```suggestion
    def _ move_from_inflight(self, in_flight_key: str):
```

Reply:
I had it like this originally but was having issues testing it. Will move it to internal function if I can get the test_expire working 👍